api/environments/sdk/serializers.py

```diff
@@ -61,6 +62,45 @@ def _get_identity(self, identifier):
 class SDKBulkCreateUpdateTraitSerializer(SDKCreateUpdateTraitSerializer):
     trait_value = TraitValueField(allow_null=True)
 
+    class Meta(SDKCreateUpdateTraitSerializer.Meta):
+        class BulkTraitListSerializer(serializers.ListSerializer):
```

Comment:
I don't love that this class is defined in the `Meta` class but it's very specific to this serializer so I didn't really want to globally define it. Open to suggestions on how to improve this though.

Reply:
It does look a bit odd, but I can't really think of a better way